examples/common.cpp

```diff
@@ -656,93 +670,60 @@ bool read_wav(const std::string & fname, std::vector<float>& pcmf32, std::vector
                 if (n == 0) {
                     break;
                 }
-                wav_data.insert(wav_data.end(), buf, buf + n);
+                audio_data.insert(audio_data.end(), buf, buf + n);
             }
         }
 
-        if (drwav_init_memory(&wav, wav_data.data(), wav_data.size(), nullptr) == false) {
-            fprintf(stderr, "error: failed to open WAV file from stdin\n");
-            return false;
-        }
-
-        fprintf(stderr, "%s: read %zu bytes from stdin\n", __func__, wav_data.size());
+	if ((result = ma_decoder_init_memory(audio_data.data(), audio_data.size(), &decoder_config, &decoder)) != MA_SUCCESS) {
+	    fprintf(stderr, "Error: failed to open audio data from stdin (%s)\n", ma_result_description(result));
+	    return false;
+	}
+	fprintf(stderr, "%s: read %zu bytes from stdin\n", __func__, audio_data.size());
     }
     else if (is_wav_buffer(fname)) {
-        if (drwav_init_memory(&wav, fname.c_str(), fname.size(), nullptr) == false) {
-            fprintf(stderr, "error: failed to open WAV file from fname buffer\n");
-            return false;
-        }
+	if ((result = ma_decoder_init_memory(audio_data.data(), audio_data.size(), &decoder_config, &decoder)) != MA_SUCCESS) {
+	    fprintf(stderr, "Error: failed to open audio data from fname buffer (%s)\n", ma_result_description(result));
+	    return false;
+	}
     }
-    else if (drwav_init_file(&wav, fname.c_str(), nullptr) == false) {
+    else if ((result = ma_decoder_init_file(fname.c_str(), &decoder_config, &decoder)) != MA_SUCCESS) {
 #if defined(WHISPER_FFMPEG)
-        if (ffmpeg_decode_audio(fname, wav_data) != 0) {
-            fprintf(stderr, "error: failed to ffmpeg decode '%s' \n", fname.c_str());
-            return false;
+        if (ffmpeg_decode_audio(fname, audio_data) != 0) {
+	    fprintf(stderr, "error: failed to ffmpeg decode '%s'\n", fname.c_str());
+	    return false;
         }
```

Comment:
Fix the indentations in a few more places in this file.

```suggestion
        if (ffmpeg_decode_audio(fname, audio_data) != 0) {
	        fprintf(stderr, "error: failed to ffmpeg decode '%s'\n", fname.c_str());
	        return false;
        }
```